Python/Tests/Core/DebugReplEvaluatorTests.cs

```diff
@@ -63,6 +65,7 @@ public class DebugReplEvaluatorTests {
             _window = new MockReplWindow(_evaluator);
             _evaluator._Initialize(_window);
             _processes = new List<PythonProcess>();
+            _moduleFileExtension = Version.Version < Microsoft.PythonTools.Parsing.PythonLanguageVersion.V30 && Version.IsCPython ? ".pyc" : ".py";
```

Comment:
After looking at the code that makes use of the module file (called from command handler for `PkgCmdIDList.cmdidOpenInteractiveScopeInEditor`), I think I'll go with the ChangeExtension(null).
```
        private static readonly IEnumerable<string> SourceExtensions = new[] {
            PythonConstants.FileExtension,
            PythonConstants.WindowsFileExtension,
        };

        private string GetCurrentScopeSourcePath() {
            var path = (_interactive.Evaluator as IMultipleScopeEvaluator)?.CurrentScopePath;
            if (string.IsNullOrEmpty(path)) {
                return null;
            }

            foreach (var ext in SourceExtensions) {
                var source = Path.ChangeExtension(path, ext);
                if (File.Exists(source)) {
                    return source;
                }
            }

            return null;
        }
```